Update patternfly-next to v1.0.109, About Modal updates

dtaylor113 opened the pull request:
To incorporate changes upstream in the About Modal.  `trademark` no longer required in `<PfAboutModal />`.  Updated `_overrides.scss`. 

![image](https://user-images.githubusercontent.com/12733153/50309304-588b1600-046c-11e9-9aa8-8831e6db1802.png)

Related PRs:
https://github.com/openshift/console/pull/936#discussion_r241780200

dtaylor113 commented:
Hmm...I'm not seeing Project Details, Status Dashboard:
![image](https://user-images.githubusercontent.com/12733153/50310199-46f73d80-046f-11e9-9161-565d01b489a8.png)

alecmerdler commented:
jenkins rebuild

priley86 commented:
per discussion w/ @mcoker, we can replace all instances of  `.pf-l-page__main-section` with `.pf-c-page__main-section` as part of this upgrade (the class was renamed).

So updating the following in `_layout.scss`:
```
html,
body,
#app,
#content,
#content-scrollable,
.pf-l-page__main-section {
  height: 100%; // so .co-p-has-sidebar, .yaml-editor are full height
}
.pf-l-page__main-section {
  --pf-global--FontSize--md: #{$font-size-base};
}
```
to:
```
html,
body,
#app,
#content,
#content-scrollable,
.pf-c-page__main-section {
  height: 100%; // so .co-p-has-sidebar, .yaml-editor are full height
}
.pf-c-page__main-section {
  --pf-global--FontSize--md: #{$font-size-base};
}
```
should resolve the missing Status page contents...

spadgett commented:
/assign @rhamilto

rhamilto commented:
Please also remove https://github.com/openshift/console/blob/02ba4c7dda385bd41f88efc565b7dbf932c4b2c4/frontend/public/components/_about-modal.scss#L9-L13 as the overrides are no longer needed now that trademark and logo are optional props.

As a follow-on, we should add the Red Hat logo to the OpenShift about modals.

dtaylor113 commented:
Implemented @priley86 and @rhamilto 's fixes,  Project status details are now displaying correctly.  About Modal seems ok as well.  I removed the `trademark` property.

dtaylor113 commented:
jenkins rebuild

dtaylor113 commented:
jenkins build

dtaylor113 commented:
jenkins rebuild

rhamilto commented:
/lgtm

dtaylor113 commented:
jenkins rebuild

spadgett commented:
jenkins rebuild

spadgett commented:
@dtaylor113 fyi, needs rebase

dtaylor113 commented:
jenkins rebuild

dtaylor113 commented:
jenkins rebuild
